snuba/query/allocation_policies/__init__.py

```diff
@@ -221,16 +221,40 @@ def runtime_config_prefix(self) -> str:
 
     @property
     def is_active(self) -> bool:
-        return bool(self.get_config_value(IS_ACTIVE))
+        try:
+            return bool(self.get_config_value(IS_ACTIVE))
+        except Exception:
+            logger.exception(
+                "Something went wrong getting a config value for an Allocation Policy."
+            )
+            if settings.RAISE_ON_ALLOCATION_POLICY_FAILURES:
+                raise
+            return True
 
     @property
     def is_enforced(self) -> bool:
-        return bool(self.get_config_value(IS_ENFORCED))
+        try:
+            return bool(self.get_config_value(IS_ENFORCED))
+        except Exception:
+            logger.exception(
+                "Something went wrong getting a config value for an Allocation Policy."
+            )
+            if settings.RAISE_ON_ALLOCATION_POLICY_FAILURES:
+                raise
+            return False
 
     @property
     def throttled_thread_number(self) -> int:
         """Number of threads any throttled query gets assigned."""
-        return int(self.get_config_value(THROTTLED_THREAD_NUMBER))
+        try:
+            return int(self.get_config_value(THROTTLED_THREAD_NUMBER))
```

Comment:
you're duplicating this code over and over. Why not `try/accept` `get_config_value`?

Reply:
These all had different return values so try/excepting the get_config_value function would leave us with an unknown value to return. I removed these checks anyhow since they aren't necessary, the main try/excepts in the quota balance methods take care of it.